abouts/socialmap.md

```diff
@@ -60,10 +60,10 @@
 
 ## שותפים במיזם {supporters}
 
-#### [![הסדנא לידע ציבורי](assets/img/hasadna.png =120x120)](https://hasadna.org.il)  [![מידות](assets/img/midot.svg =120x72)](http://www.midot.org.il/)   [![חברת מגדל](assets/img/migdal.svg =120x72)](https://www.migdal.co.il/)
```

Comment:
why?

Reply:
what do you mean by "why"? these are our partners... isn't it so?

Reply:
@akariv did not you tell me we should add #### to centralise the icons on this page?